flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1722,6 +1723,28 @@ def test_get_device_path_unattached_volume(self):
         )
         self.assertEqual(new_volume.blockdevice_id, exception.blockdevice_id)
 
+    def test_get_device_path_detached_volume(self):
+        """
+        ``get_device_path`` raises ``UnattachedVolume`` if the supplied
+        ``blockdevice_id`` corresponds to a volume that was attached to the
+        node but has been detached from it.
+        """
```

Comment:
No change needed.

Out of curiosity, was this test failing before the changes to ebs?